### PR TITLE
Publish to Marketplace content

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | cd74d1e |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-261.amplify-central.opendocs-builder.pcloud.axway.int |

**Comment by lbadenhop** [comment]
@georgiana-caras, please approve content before I review and merge. Thanks.

File: content/en/docs/manage_service_registry/service_management.md [hunk 1]
```diff
@@ -74,3 +75,18 @@ View the following detailed information for a specific service in any state (Dra
 3. Click on the environment name breadcrumb at the top of the page (i.e., the name to the right of **ENVIRONMENTS / NAME**).
 4. In the *Services* tab at the bottom of the page, click on the trash bin associated with the service you want to delete.
 5. Type the name of the service and click on the confirmation button that “I understand this is a permanent and irreversible action. Delete.” **All versions of the service are deleted.**
+
+## Publish to the Marketplace
```
Comment (georgiana-caras) [inline content/en/docs/manage_service_registry/service_management.md:79]: Publish to Marketplace from the Service Registry enables providers to quickly promote an API to the Marketplace for internal consumption as a product under a Free plan. An asset and a product will automatically be generated. The publication is initially restricted to teams in the provider organization, but changes can be made to the product from the Product Foundry at any point in time.

File: content/en/docs/manage_service_registry/service_management.md [hunk 2]
```diff
@@ -74,3 +75,18 @@ View the following detailed information for a specific service in any state (Dra
...
+3. Select **Publish to Marketplace**.
+4. A sideblade will be displayed and you can add / edit information where applicable:
+
+    * Asset Name - The proposed asset name is set to the API Service name.  
+    * Product Name - The proposed asset name is set to the API Service name.
+    * Team Ownership - The proposed team ownership is set to the currently logged in team.  If you are logged in as a Central Admin, no team ownership is proposed and the created Asset and Product are only visible to the Central Admin role.
+    * Product Visibility - The proposed visibility is set to the currently logged in team to facilitate collaboration with internal teams to edit the product documentation or product plan.  If you are logged in as a Central Admin, the proposed Product Visibility is set to none.
+    * The default product plan will use a free unlimited plan which can be edited after the product is created.
+
+5. Click **Publish** to quickly make an API service available from consumption in the Marketplace.
```
Comment (georgiana-caras) [inline content/en/docs/manage_service_registry/service_management.md:92]: API Service available for consumption

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

**Pull request merged**